driver/cmd/ibm-spectrum-scale-csi/main.go

```diff
@@ -191,12 +195,14 @@ func InitFileLogger() func() {
 	_, err := os.Stat(filePath)
 	if os.IsNotExist(err) {
 		fileDir, _ := path.Split(filePath)
+		/* #nosec */
```

Comment:
It will be good to mention the go security rule and reason for skipping gosec scane, while adding a nosec comment here, similar to the way it is handled in other places, e.g. 
```
// #nosec G101 false positive
```

Reply:
Done